companion/src/firmwares/opentx/opentxeeprom.cpp

```diff
@@ -1369,13 +1369,13 @@ class InputField: public TransformedField {
 
     virtual void beforeExport()
     {
-      _weight = smallGvarToEEPROM(expo.weight);
+      _weight = smallGvarImport(expo.weight);
 
-      if (IS_HORUS_OR_TARANIS(board) && version >= 216) {
-        _offset = smallGvarToEEPROM(expo.offset);
+      if ((IS_HORUS_OR_TARANIS(board) && version >= 216) || (IS_ARM(board) && version >= 218)) {
+        _offset = smallGvarImport(expo.offset);
       }
 
-      if (!IS_TARANIS(board) || version < 216) {
+      if ((!IS_ARM(board) && version < 218) || (!IS_TARANIS(board) && version < 217)|| version < 216) {
```

Comment:
I think the right thing here is 
```
      if ((!IS_ARM(board) && version >= 218) || (!IS_TARANIS(board) && version == 217)|| version < 216) {
```

That also changes Horus in 218. But my quick test shows that the import is right and I would be very suprised if horus is different from other arm platforms in 2.2